etc/replace-strings-in-doc.py

```diff
@@ -53,6 +53,7 @@ def dive(path):
     "ImageLeftActionPPerm16PPerm16": "ImageLeftAction",
     "ImageRightActionPPerm16PPerm16": "ImageRightAction",
     "aho_corasick_dot.*": "",
+    "word_graph_dot.*": "Returns a Dot object representing a word graph.",
```

Comment:
If #190 is merged, there may be a slight merge conflict here. The solution would be to delete the `"aho_corasick_dot.*": "",` line, and move `"word_graph_dot.*": "Returns a Dot object representing a word graph.",`  from this dictionary to the new dictionary called `docstring_replacements` in `conf.py`. Sorry for creating more work!